tad/models/task.py

```diff
@@ -0,0 +1,30 @@
+from pydantic import BaseModel, ValidationInfo, field_validator
+from pydantic import Field as PydanticField  # type: ignore
+from sqlmodel import Field as SQLField  # type: ignore
+from sqlmodel import SQLModel
+
+
+class Task(SQLModel, table=True):
+    id: int = SQLField(default=None, primary_key=True)
+    title: str
+    description: str
+    sort_order: float
+    status_id: int | None = SQLField(default=None, foreign_key="status.id")
+    user_id: int | None = SQLField(default=None, foreign_key="user.id")
+    # todo(robbert) Tasks probably are grouped (and sub-grouped), so we probably need a reference to a group_id
+
+
+class MoveTask(BaseModel):
+    # todo(robbert) values from htmx json are all strings, using type int does not work for
+    #  sibling variables (they are optional)
+    id: str = PydanticField(None, alias="taskId", strict=False)
```

Comment:
YOu might want to impelment this for aliases: https://docs.pydantic.dev/latest/concepts/alias/#using-alias-generators

Reply:
Worth considering (for another time).